planner/cascades/testdata/transformation_rules_suite_out.json

```diff
@@ -2456,5 +2456,286 @@
+          "    Window_8 input:[Group#2]",
+          "Group#2 Schema:[test.t.a,test.t.b,test.t.c,test.t.d,Column#14]",
+          "    Window_4 input:[Group#3]",
+          "Group#3 Schema:[test.t.a,test.t.b,test.t.c,test.t.d]",
+          "    DataSource_1 table:t"
+        ]
+      },
+      {
+        "SQL": "select a, b, rank() over (), max(b) over (partition by c), c, d, dd from (select a, b, c, d, sum(d) over (partition by c) as dd from t) as tt",
+        "Result": [
+          "Group#0 Schema:[test.t.a,test.t.b,Column#18,Column#17,test.t.c,test.t.d,Column#14]",
+          "    Projection_11 input:[Group#1], test.t.a, test.t.b, Column#18, Column#17, test.t.c, test.t.d, Column#14",
+          "Group#1 Schema:[test.t.a,test.t.b,test.t.c,test.t.d,Column#14,Column#17,Column#18]",
+          "    Window_10 input:[Group#2]",
+          "Group#2 Schema:[test.t.a,test.t.b,test.t.c,test.t.d,Column#14,Column#17]",
+          "    Window_12 input:[Group#3]",
+          "Group#3 Schema:[test.t.a,test.t.b,test.t.c,test.t.d]",
+          "    DataSource_1 table:t"
+        ]
+      },
+      {
+        "SQL": "select a, b, sum(bb) over (partition by a) as 'sum_bb' from (select a, b, max(b) over (partition by a) as 'bb' from t) as tt",
+        "Result": [
+          "Group#0 Schema:[test.t.a,test.t.b,Column#19]",
+          "    Projection_11 input:[Group#1], test.t.a, test.t.b, Column#16",
+          "Group#1 Schema:[test.t.a,test.t.b,Column#14,Column#16]",
+          "    Window_8 input:[Group#2]",
+          "Group#2 Schema:[test.t.a,test.t.b,Column#14]",
+          "    Window_4 input:[Group#3]",
+          "Group#3 Schema:[test.t.a,test.t.b]",
+          "    DataSource_1 table:t"
+        ]
+      },
+      {
+        "SQL": "select a, b, sum(bb) over (partition by a) as 'sum_bb', c from (select a, b, c, max(b) over (partition by a) as 'bb' from t) as tt",
+        "Result": [
+          "Group#0 Schema:[test.t.a,test.t.b,Column#19,test.t.c]",
+          "    Projection_11 input:[Group#1], test.t.a, test.t.b, Column#16, test.t.c",
+          "Group#1 Schema:[test.t.a,test.t.b,test.t.c,Column#14,Column#16]",
+          "    Window_8 input:[Group#2]",
+          "Group#2 Schema:[test.t.a,test.t.b,test.t.c,Column#14]",
+          "    Window_4 input:[Group#3]",
+          "Group#3 Schema:[test.t.a,test.t.b,test.t.c]",
+          "    DataSource_1 table:t"
+        ]
+      },
+      {
+        "SQL": "select a, b, rank() over (partition by a), sum(bb) over (partition by a) as 'sum_bb', c from (select a, b, c, max(b) over (partition by a) as 'bb' from t) as tt",
+        "Result": [
+          "Group#0 Schema:[test.t.a,test.t.b,Column#21,Column#22,test.t.c]",
+          "    Projection_13 input:[Group#1], test.t.a, test.t.b, Column#18, Column#17, test.t.c",
+          "Group#1 Schema:[test.t.a,test.t.b,test.t.c,Column#17,Column#18]",
+          "    Window_10 input:[Group#2]",
```

Comment:
It's difficult to figure out which window function this plan is corresponding to from this result.

I think we'd better add some detailed information about the window operator in the explained result, e.g. the window function name and the partition columns. 

Perhaps we can fix this in another PR.